web-marketplace/src/components/organisms/UserSellOrders/UserSellOrders.Toolbar.tsx

```diff
@@ -47,25 +49,37 @@ export const UserSellOrdersToolbar = ({
         <Subtitle size="xl" className="text-base sm:text-[22px] pt-3">
           <Trans>Sell orders</Trans>
         </Subtitle>
-        {hasTradableCredits ? (
-          <CreateButton
-            hasTradableCredits={hasTradableCredits}
-            setIsSellFlowStarted={setIsSellFlowStarted}
-          />
-        ) : (
-          <InfoTooltip
-            arrow
-            placement="top"
-            title={_(msg`You have no tradable credits that can be sold.`)}
-          >
-            <div>
+        <div className="flex sm:flex-row flex-col sm:items-center items-end">
+          {loginDisabled && (
```

Comment:
we would need to check for `privActiveAccount.can_use_stripe_connect` too because only users with this enabled can create USD sell orders normally... but it won't be defined when `loginDisabled` is true, so it's actually the same issue as above...